### PR TITLE
Fix the issue in generating swagger when WSDL has inline nested complex types definitions 

### PR DESCRIPTION
**Pull request by vithu30**
This PR fixes wso2/product-apim#7108. Ports the fix added to 2.6.0

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/wsdl/WSDL11SOAPOperationExtractor.java [hunk 1]
```diff
@@ -386,11 +388,55 @@ private void addModelDefinition(Node current, ModelImpl model, String type, bool
...
+                                                    ((ArrayProperty) parentProperty).setItems(objectProperty);
+                                                    parentProp = parentProperty;
+                                                }
+                                            } else {
+                                                ((ArrayProperty) parentProperty)
+                                                        .setItems(createPropertyFromNode(current, true));
+                                            }
+                                        } else if (parentProperty instanceof ObjectProperty) {
+                                            if (((ObjectProperty) parentProperty).getProperties() != null) {
+                                                if ((((ObjectProperty) parentProperty).getProperties()).get(parentName) != null) {
```
Comment (ruks) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/wsdl/WSDL11SOAPOperationExtractor.java]: Please check whether the length is exceeded?

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/wsdl/WSDL11SOAPOperationExtractor.java [hunk 2]
```diff
@@ -448,6 +522,21 @@ private void addModelDefinition(Node current, ModelImpl model, String type, bool
...
 
+    private boolean isChildNode(Node current, Node prevNode) {
+        boolean isChild = false;
+        if (prevNode.hasChildNodes()) {
+            NodeList childNodes = prevNode.getChildNodes();
+            for (int i = 0; i < childNodes.getLength(); i++) {
+                if (childNodes.item(i).getAttributes() != null && current.getAttributes()
+                        .getNamedItem(SOAPToRESTConstants.NAME_ATTRIBUTE).getNodeValue().equals(childNodes.item(i)
+                                .getAttributes().getNamedItem(SOAPToRESTConstants.NAME_ATTRIBUTE).getNodeValue())) {
+                    isChild = true;
```
Comment (ruks) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/wsdl/WSDL11SOAPOperationExtractor.java]: can we just return true? If it is true, no need to loop remaining objects right?

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/wsdl/WSDL11SOAPOperationExtractor.java (hunk 1 above)
Comment (vithu30) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/wsdl/WSDL11SOAPOperationExtractor.java]: Fixed

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/wsdl/WSDL11SOAPOperationExtractor.java (hunk 2 above)
Comment (vithu30) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/wsdl/WSDL11SOAPOperationExtractor.java]: Yes..Fixed.

**Pull request merged**